File: app/src/main/java/com/parametris/iteng/asdf/track/LokAlarmReceiver.java
package com.parametris.iteng.asdf.track;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class LokAlarmReceiver extends BroadcastReceiver {
    public LokAlarmReceiver() {
    }

    @Override
    public void onReceive(Context context, Intent intent) {
        context.startService(new Intent(context, LokService.class));
    }
}

File: app/src/main/java/com/parametris/iteng/asdf/track/LokAlarmReceiver.kt
package com.parametris.iteng.asdf.track

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class LokAlarmReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        context.startService(Intent(context, LokService::class.java))
    }
}
